docs/source/api/v3/servers.rst

```diff
@@ -36,7 +36,9 @@ Request Structure
 	+============+==========+===================================================================================================================+
 	| cachegroup | no       | Return only those servers within the :term:`Cache Group` that has this :ref:`cache-group-id`                      |
 	+------------+----------+-------------------------------------------------------------------------------------------------------------------+
-	| dsId       | no       | Return only those servers assigned to the :term:`Delivery Service` identified by this integral, unique identifier |
+	| dsId       | no       | Return only those servers assigned to the :term:`Delivery Service` identified by this integral, unique identifier.|
+	|            | no       | If the Delivery Service has a :term:` Topology` assigned to it, the :ref:`to-api-servers` endpoint will return    |
```

Comment:
The space in
```rst
:term:` Topology`
```
is causing it to render improperly

Reply:
Removed space in 348c20da2